tests/integration/rules/vatin.phpt

```diff
@@ -8,11 +9,18 @@ declare(strict_types=1);
 
 require_once 'vendor/autoload.php';
 
+use Respect\Validation\Exceptions\ComponentException;
 use Respect\Validation\Exceptions\Locale\PlVatinException;
 use Respect\Validation\Exceptions\NestedValidationException;
 use Respect\Validation\Exceptions\VatinException;
 use Respect\Validation\Validator as v;
 
+try {
+    v::vatin('BR')->check('1645865778');
```

Comment:
This exception is not part of an integration test. Could you restore the unit test?

Reply:
ok @henriquemoody , I thought this test was part of the integration test